feature-staking-impl/src/main/java/jp/co/soramitsu/feature_staking_impl/presentation/staking/alerts/model/AlertModel.kt

```diff
@@ -0,0 +1,38 @@
+package jp.co.soramitsu.feature_staking_impl.presentation.staking.alerts.model
+
+import android.view.View
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import jp.co.soramitsu.feature_staking_impl.R
+import jp.co.soramitsu.feature_staking_impl.domain.Alert
+import jp.co.soramitsu.feature_staking_impl.presentation.StakingRouter
+
+class AlertModel(
+    @DrawableRes val icon: Int,
+    @StringRes val title: Int,
+    @StringRes val extraMessage: Int,
+    val startFlow: ((View) -> Unit)? = null,
+) {
+    val isWarning: Boolean = startFlow == null
```

Comment:
Make sealed structure.
Move all logic to mapper

Reply:
Done